examples/hacker_news_pwa/lib/hacker_news_service.dart

```diff
@@ -1,28 +1,27 @@
 import 'dart:async';
 import 'dart:convert';
+import 'dart:html';
 
 import 'package:angular/angular.dart';
-import 'package:http/http.dart';
 
 /// Represents the base URL for HTTP requests using [HackerNewsService].
 const baseUrl = const OpaqueToken<String>('baseUrl');
 const defaultBaseUrl = 'https://node-hnapi.herokuapp.com';
 
 class HackerNewsService {
   final String _baseUrl;
-  final BaseClient _client;
 
-  HackerNewsService(@baseUrl this._baseUrl, this._client);
+  HackerNewsService(@baseUrl this._baseUrl);
```

Comment:
We need to be able to write unit tests for this code, so the HTTP service needs to be injectable or otherwise overridable for testing. Unless we've decided pkg/http shouldn't be used anymore because of code bloat I'd rather continue to push on getting the size down there.

Reply:
We can always create a trivial, injectable class.

I can't imagine a way to make `pkg:http` smarter in the trivial case.

There is **so** much plumbing between newing up a client and hitting send – to expect dart2js to figure out that at runtime we only need trivial `get` – no headers, no peeking at response status code – I think it's unrealistic.

Reply:
Fair enough. I merged this change into my PR (#1084).